packages/cli/src/constructs/api-check.ts

```diff
@@ -241,6 +257,11 @@ export class ApiCheck extends Check {
   localTearDownScript?: string
   degradedResponseTime?: number
   maxResponseTime?: number
+  private readonly setupScriptDependencies?: Array<ScriptDependency>
+  private readonly tearDownScriptDependencies?: Array<ScriptDependency>
+  private readonly setupScriptPath?: string
```

Comment:
We should make this like BCR IMO, like content or entrypoint. That way, they are identical

Reply:
I'm unsure how to rewrite this while staying backwards compatible.